processor/src/lib.rs

```diff
@@ -181,10 +181,8 @@ pub fn execute(
     advice_inputs: AdviceInputs,
     host: &mut impl SyncHost,
     options: ExecutionOptions,
-    source_manager: Arc<dyn SourceManager>,
 ) -> Result<ExecutionTrace, ExecutionError> {
-    let mut process = Process::new(program.kernel().clone(), stack_inputs, advice_inputs, options)
-        .with_source_manager(source_manager);
+    let mut process = Process::new(program.kernel().clone(), stack_inputs, advice_inputs, options);
```

Comment:
We should also remove the source manager from `FastProcessor`

Reply:
I am surprised `make build` passed :monocle_face: